recipes/prismatic/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
This shebang is not necessary, could you please remove it?
```suggestion
```

Reply:
Yep - done (as part of a general tidy up commit)